packages/apps/plugins/plugin-status-bar/src/components/StatusBar.tsx

```diff
@@ -27,7 +27,10 @@ const StatusBarButton = forwardRef<HTMLButtonElement, StatusBarButtonProps>(
       <Component
```

Comment:
I know this isn’t part of the PR, but generally this should be achieved using Radix-style `asChild` and `Slot`, just noting for future reference.

Reply:
Hmm. I'm using the radix Slot component. I still haven't internalised the pattern yet.

Reply:
Ah, nevermind, yes you are! Generally the variable is named `Root` in this case, nbd.

Reply:
```ts
const StatusBarButton = forwardRef<HTMLButtonElement, StatusBarButtonProps>(
  ({ classNames, children, asChild, ...props }, forwardedRef) => {
    const classes = mx(
      classNames,
      'flex items-center gap-2 p-1 px-2 rounded-sm',
      'select-none cursor-pointer',
      'hover:bg-neutral-75 active:bg-neutral-150',
      'dark:hover:bg-neutral-750 dark:active:bg-neutral-700',
    );

    return asChild ? (
      <Slot role='button' ref={forwardedRef} className={classes}>
        {children}
      </Slot>
    ) : (
      <button className={classes} ref={forwardedRef} {...props}>
        {children}
      </button>
    );
  },
);
```

Like this?

Reply:
No, just rename `Component` to `Root` and it’ll be consistent with Radix and our other components that do this.

Reply:
I was just confused because we sometimes pass a component to use for rendering mosaic tiles as a prop named `{…}Component` and assumed without looking at the context that that’s what was going on, welcome to ignore this whole thread 😅.

Reply:
Now that I'm here I like the ternary better. ¯\_(ツ)_/¯.

Noted on the naming convention though!